.travis.yml

```diff
@@ -1,34 +1,33 @@
-# See https://github.com/silverstripe/silverstripe-travis-support for setup details
 language: php
 
-sudo: false
-
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.0.x-dev
-    - CORE_RELEASE=4
-    # The path of the module when installed by composer
-    - MODULE_PATH=iframe
+    - COMPOSER_ROOT_VERSION=5.0.x-dev
 
 matrix:
-  fast_finish: true
   include:
     - php: 5.6
-      env: DB=MYSQL
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
-      env: DB=MYSQL
+      env: DB=MYSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
 
 before_script:
-  - composer self-update || true
+  # Init PHP
   - phpenv rehash
   - phpenv config-rm xdebug.ini
+  - echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+  # Install composer dependencies
   - composer validate
-  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
-  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
-  - cd ~/builds/ss
-  - composer install --prefer-dist
+  - composer require --no-update silverstripe/recipe-cms:1.0.x-dev silverstripe-themes/simple:~3.2
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
 
 script:
-  - vendor/bin/phpunit $MODULE_PATH/tests
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
+  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs code/ tests/; fi
```

Comment:
We've started including root level files, e.g. `vendor/bin/phpcs code/ tests/ *.php`